pkgs/build-support/alternatives/blas/default.nix

```
@@ -65,7 +65,11 @@ stdenv.mkDerivation {
   installPhase = (''
   mkdir -p $out/lib $dev/include $dev/lib/pkgconfig
 
-  libblas="${lib.getLib blasProvider'}/lib/libblas${canonicalExtension}"
+  if [ "${builtins.toString stdenv.hostPlatform.isDarwin}" == "1" ] && [ "${blasImplementation}" == "mkl" ]; then
```

This is the way we usually write these conditionals in Nixpkgs:

```suggestion
  if [ -n "${builtins.toString stdenv.hostPlatform.isDarwin}" ] && [ "${blasImplementation}" == "mkl" ]; then
```